### PR TITLE
chore(dev): add reactotron for inspection

### PR DESCRIPTION
**Pull request by patw0929**
Please refer this project: [reactotron](https://github.com/infinitered/reactotron).

Reactotron is cross-platform react/react-native inspector.
It might be helpful when developing. We can see not only the redux actions, store structure & content, but also the API response (including response content/header & request header) in our app.

Screenshots:

- API Response:

(If we want to see the network response content, it need to turn off the debug mode. Seems their bug.)

- store:

### CODE REVIEW

**Comment by lex111** [comment]
Wow, cool, I always wanted to try it, now there will be an opportunity!

**Comment by Antoine38660** [comment]
Seems really cool! I never used it too, so good opportunity yes.

Is not it going to slow down the execution in dev mode? I know we had to remove the `redux-devtools` for this reason.

**Comment by patw0929** [comment]
The `reactotron` is not like `remote-redux-devtools` which is running in browser (Chrome extensions), it is a standalone app.
And I think the speed is ok for me. Could anybody help to try it? 😄 Thanks!

File: package.json [hunk 1]
```diff
@@ -67,6 +67,8 @@
     "react-navigation": "^1.0.0-beta.11",
     "react-redux": "^5.0.2",
     "react-syntax-highlighter": "^5.6.2",
+    "reactotron-react-native": "^1.12.2",
+    "reactotron-redux": "^1.12.2",
```
Comment (housseindjirdeh) [inline package.json]: Probably makes more sense to have both as part of `devDependencies`

**Comment by housseindjirdeh** [comment]
Oooooooh I've stumbled on `reactotron` before and it looked awesome :O

Will pull this down and give it a shot tonight. If everyone's okay with having it included in our app as part of our development workflow - I'm okay with that as well :)

I would also like to possibly have it as part of a separate script command similar to the amazing work @lex111 did with `logger`. Something like `yarn:start:ios:tron` to run boot the application with reactotron based on a environment variable. What do you think?

**Comment by Antoine38660** [comment]
I totally agree with you, a specific  command like `yarn:start:ios:tron` is a very good idea

**Comment by patw0929** [comment]
Good idea. 👍  I have added the `yarn:start:ios:tron` & `yarn:start:android:tron` command in `packages.json`.

**Comment by housseindjirdeh** [comment]
Awesome thanks a million matey :+1: 

Excited to try this out, merging this beauty in :)

**Pull request merged**